.readthedocs.yaml

```diff
@@ -0,0 +1,31 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.10"
+    
+# Build documentation in the "docs/" directory with Sphinx
+sphinx:
+  configuration: doc_template/source/conf.py
+
+# Optionally build your docs in additional formats such as PDF and ePub
+# formats:
+#    - pdf
+#    - epub
+
+# Optional but recommended, declare the Python requirements required
+# to build your documentation
+# See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
+python:
+  install:
+    - method: pip
+      path: .
+      extra_requirements:
+        - dev
```

Comment:
We should discriminate the docs dependencies as a new section in the project.toml. This is especially important given the reliance on erdantic.

Additionally, if we are using erdantic, i think the action will need to install graphviz
https://github.com/marketplace/actions/setup-graphviz